NeuroID/NeuroIDClass/Extensions/NIDSend.swift

```diff
@@ -128,6 +128,10 @@ internal extension NeuroID {
         }, onFailure: { error in
             logError(category: "APICall", content: String(describing: error))
         })
+        
+        if NeuroID.isLowMemory() {
+            NeuroID.forcePauseCollection()
+        }
```

Comment:
I don't think this is necessary, we should be handling the same as android where we don't officially "pause" collection but just drop events

Reply:
In pause colleciton we do other operations that require memory, such as attempt to do a groupAndPost event again. I'm trying to minimize the amount of memory operations in the event of low memory

Reply:
Right, I am saying we don't need to call pause collection at all. We just drop events as they come in. If we pause it because of low memory, how would we re-enable it/how would the customer know to re-enable it?